### PR TITLE
update docs for new release

### PR DESCRIPTION
**Pull request by Despire**
Changelog for 0.9.0 release

### CODE REVIEW

**Review by JKBGIT1 — APPROVED** [review]
LGTM :+1:

File: docs/CHANGELOG/changelog-0.9.x.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Claudie `v0.9`
+
+!!! warning "Due to changes to the core of how claudie works with terraform files and representation of the data in persistent storage the `v0.9.x` version will not be backwards compatible with clusters build using previous claudie versions."
```
Comment (bernardhalas) [inline docs/CHANGELOG/changelog-0.9.x.md:3]: `s/build/built/`

File: docs/CHANGELOG/changelog-0.9.x.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+## Most notable changes (TL;DR)
+
+- Support for pluggable external terraform files was added, breaking the dependency of updated terraform files on a new claudie version. The ability to arbitrarily change the templates used by claudie was made available to the user. As a result, claudie has implemented a rolling update of the infrastructure in case a change in the terraform templates is detected, by gradually updating the build cluster one nodepool at a time.
```
Comment (bernardhalas) [inline docs/CHANGELOG/changelog-0.9.x.md:7]: `s/claudie/Claudie/`
`s/hetzner/Hetzner/`
`s/ubuntu/Ubuntu/`

Throughout the whole doc.

**Review by bernardhalas — COMMENTED** [review]
Just a few typos seen.

**Pull request merged**